src/hooks/login/useLedgerLogin.ts

```diff
@@ -341,7 +342,7 @@ export const useLedgerLogin = ({
         return onLoginFailed(failInitializeErrorText);
       }
 
-      if (accounts.length === 0 || startIndex > 0) {
+      if (accounts.length === 0 || startIndex >= 0) {
```

Comment:
why the change?

Reply:
Because when navigating to the second page of the table and then going back to the first, it'd no longer fetch the first page of addresses, and instead it'd show the ones from the second page. This can be replicated even now, on the live version.